mod-orders/schemas/patch_pieces_request.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Patch pieces request",
+  "type": "object",
+  "properties": {
+    "pieces": {
```

Comment:
propose to rename to pieceIds

Reply:
in many places we use the plural form to specify the set of uuid